include/radio_types.h

```diff
@@ -994,7 +1034,7 @@ typedef struct radio_cell_info_1_4 {
         RadioCellInfoNr nr RADIO_ALIGNED(8);
     } info RADIO_ALIGNED(8);
 } RADIO_ALIGNED(8) RadioCellInfo_1_4; /* Since 1.2.5 */
-G_STATIC_ASSERT(sizeof(RadioCellInfo_1_4) == 128);
+G_STATIC_ASSERT(sizeof(RadioCellInfo_1_4) == 136);
```

Comment:
Good catch!

Reply:
Apparently I confused `::android::hardware::radio::V1_4::CellInfo` with `::android::hardware::radio::V1_4::CellInfo::Info` 😐

Reply:
It is so easy to do.